addon/configuration.js

```diff
@@ -97,5 +97,17 @@ export default {
         this[property] = getWithDefault(config, property, DEFAULTS[property]);
       }
     }
+  },
+
+  get(keyName) {
+    if (keyName === 'baseURL') {
+      deprecate(
+        `Ember Simple Auth: Reading the property 'baseURL' is deprecated. It will become private beginning version 2.0.0.`,
+        false,
+        { id: 'ember-simple-auth.configuration.baseURL', until: '2.0.0' }
+      );
+    }
+
+    return this._super(...arguments);
```

Comment:
We should actually make the property defined in line 41 a [deprecating alias](http://emberjs.com/api/classes/Ember.computed.html#method_deprecatingAlias) for a to-be-added rootURL property.